types/stripe-js/elements/address.d.ts

```diff
@@ -0,0 +1,243 @@
+  ): StripeAddressElement;
+
+  /**
+   * Triggered when the element is fully rendered and can accept `element.focus` calls.
+   */
+  on(
+    eventType: 'ready',
+    handler: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+  once(
+    eventType: 'ready',
+    handler: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+  off(
+    eventType: 'ready',
+    handler?: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+
+  /**
+   * Triggered when the element gains focus.
+   */
+  on(
+    eventType: 'focus',
+    handler: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+  once(
+    eventType: 'focus',
+    handler: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+  off(
+    eventType: 'focus',
+    handler?: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+
+  /**
+   * Triggered when the element loses focus.
+   */
+  on(
+    eventType: 'blur',
+    handler: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+  once(
+    eventType: 'blur',
+    handler: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+  off(
+    eventType: 'blur',
+    handler?: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+
+  /**
+   * Triggered when the escape key is pressed within the element.
+   */
+  on(
+    eventType: 'escape',
+    handler: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+  once(
+    eventType: 'escape',
+    handler: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+  off(
+    eventType: 'escape',
+    handler?: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+
+  /**
+   * Triggered when the element fails to load.
+   */
+  on(
+    eventType: 'loaderror',
+    handler: (event: {elementType: 'address'; error: StripeError}) => any
+  ): StripeAddressElement;
+  once(
+    eventType: 'loaderror',
+    handler: (event: {elementType: 'address'; error: StripeError}) => any
+  ): StripeAddressElement;
+  off(
+    eventType: 'loaderror',
+    handler?: (event: {elementType: 'address'; error: StripeError}) => any
+  ): StripeAddressElement;
+
+  /**
+   * Triggered when the loader UI is mounted to the DOM and ready to be displayed.
+   */
+  on(
+    eventType: 'loaderstart',
+    handler: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+  once(
+    eventType: 'loaderstart',
+    handler: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+  off(
+    eventType: 'loaderstart',
+    handler?: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+
+  /**
+   * Updates the options the `AddressElement` was initialized with.
+   * Updates are merged into the existing configuration.
+   */
+  update(options: Partial<StripeAddressElementOptions>): StripeAddressElement;
+};
+
+export interface ContactOption {
```

Comment:
Question: should we wait until API review has been gaveled for GA to release this? What is the significance of adding `Option` to the name instead of just `Contact`?

Reply:
good question! I think it should be fine for us to add it now since we want beta users to test it out without running into type error and the whole AE is marked as "beta access" only. I was following the convention we have to append a `option` after the type name, such as [`FieldOption`](https://github.com/stripe/stripe-js/blob/c6479232/types/stripe-js/elements/payment.d.ts#L145).